packages/data-explorer/src/main.ts

```diff
@@ -19,6 +20,10 @@ router.beforeEach((to, from, next) => {
 // This way the mainView can coordinate the events
 Vue.prototype.$eventBus = new Vue()
 
+BootstrapExplorer().then((statusMessage: string | undefined) => {
+  if (statusMessage) store.dispatch('notifyUser', { type: 'success', message: statusMessage })
```

Comment:
Wellicht is dit een warning?
'Succes' nodigt mij niet uit om als admin in te loggen.